pkg/fs/fs.go

```diff
@@ -15,3 +17,31 @@ func newFileExistsError(path string) FileExistsError {
 func (e FileExistsError) Error() string {
 	return fmt.Sprintf("operation not allowed, file %q exists", e.path)
 }
+
+// DiskStatus is returned by DiskUsage
+type DiskStatus struct {
+	All   uint64
+	Used  uint64
+	Free  uint64
+	Avail uint64
+}
+
+const (
+	KB = 1024
+	MB = 1024 * KB
+	GB = 1024 * MB
+)
+
+// HumanSize formats number representing a disk space to human readable text with closest order units
+func HumanSize(size uint64) string {
+	switch {
+	case size > GB:
+		return fmt.Sprintf("%.1fGB", float64(size)/GB)
+	case size > MB:
+		return fmt.Sprintf("%.1fMB", float64(size)/MB)
+	case size > KB:
+		return fmt.Sprintf("%.1fKB", float64(size)/KB)
+	default:
+		return fmt.Sprintf("%dB", size)
+	}
+}
```

Comment:
We already have a dependency on the https://github.com/dustin/go-humanize package, so you should be able to use `humanize.Bytes`:

https://pkg.go.dev/github.com/dustin/go-humanize#Bytes